### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by squigglybob**
fixes some problems

### CODE REVIEW

File: porches/generic/site/archive.php [hunk 1]
```diff
@@ -50,15 +50,15 @@ public function dt_custom_dir_attr( $lang ){
     }
 
     public function dt_magic_url_base_allowed_js( $allowed_js ) {
-        return P4_Ramadan_Porch_Landing_Enqueue::load_allowed_scripts();
+        return DT_Generic_Porch_Landing_Enqueue::load_allowed_scripts();
```
Comment (corsacca) [inline porches/generic/site/archive.php:53]: @squigglybob this does not exist?

Comment (corsacca) [reply porches/generic/site/archive.php:53]: DT_GenericPorch_Landing_Enqueue does tho

Comment (squigglybob) [reply porches/generic/site/archive.php:53]: ah, good spot, I think when I merged thinks must have gone awry. I'll get that switched over to proper camel casing

Comment (squigglybob) [reply porches/generic/site/archive.php:53]: @corsacca should be fixed

**Pull request merged**